packages/common-scripts/src/omnilock-ethereum-displaying.ts

```diff
@@ -0,0 +1,51 @@
+import { BytesLike, bytes } from "@ckb-lumos/codec";
+import { hexify } from "@ckb-lumos/codec/lib/bytes";
+
+const COMMON_PREFIX = "CKB transaction: 0x";
+
+export interface Provider {
+  request: {
+    (payload: {
+      method: "personal_sign";
+      params: [string /*from*/, string /*message*/];
```

Comment:
It can be defined as 
```ts
params: [from: string, message: string]
```

Ref: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html#labeled-tuple-elements

Reply:
refactored